tests/test_helpers.py

```diff
@@ -0,0 +1,131 @@
+from django.conf import settings
+
+from cms.test_utils.testcases import CMSTestCase
+
+from djangocms_navigation.helpers import get_navigation_node_for_content_object
+from djangocms_navigation.test_utils import factories
+from djangocms_navigation.test_utils.polls.models import Poll, PollContent
+
+
+class NavigationContentTypeSearchTestCase(CMSTestCase):
+
+    def setUp(self):
+        self.language = settings.LANGUAGES[0][0]
+
+    def test_page_content_type_in_node_tree(self):
+        """
+        Test to ensure the correct node mapped to poll content is returned by the helper"
+        """
+        menu_contents = factories.MenuContentFactory()
+        factories.ChildMenuItemFactory(parent=menu_contents.root)
+        page_content = factories.PageContentWithVersionFactory(
+            language=self.language, version__created_by=self.get_superuser()
+        )
+        child2 = factories.ChildMenuItemFactory(parent=menu_contents.root, content=page_content)
+
+        result = get_navigation_node_for_content_object(menu_contents, page_content)
+
+        self.assertEqual(result, child2)
+
+    def test_to_search_content_object_in_nested_node_tree_search(self):
+        """
+        Test to ensure  the first node is returned by the helper if more than one node is mapped to content object
+        """
+        menu_contents = factories.MenuContentFactory()
+        child1 = factories.ChildMenuItemFactory(parent=menu_contents.root)
+        page_content = factories.PageContentWithVersionFactory(
+            language=self.language, version__created_by=self.get_superuser()
+        )
+        factories.ChildMenuItemFactory(parent=menu_contents.root)
+        grandchild = factories.ChildMenuItemFactory(parent=child1)
+        grandchild1 = factories.ChildMenuItemFactory(parent=grandchild)
+        grandchild2 = factories.ChildMenuItemFactory(parent=grandchild1, content=page_content)
+
+        result = get_navigation_node_for_content_object(menu_contents, page_content)
+
+        self.assertEqual(result, grandchild2)
+
+    def test_content_object_not_found_in_node_tree(self):
+        """
+        Test to ensure False if returned  by the helper when no node is mapped to content in node tree
```

Comment:
False is returned by the helper when no node is mapped to content in the node tree